### PR TITLE
PR: Add fix for combobox indicator and padding

### LINKED ISSUE #132: Checkboxes appear in QComboBox drop-down list
- QDarkStyle: latest
- OS: Ubuntu 16.04
- Qt: 5.5.1
- C++

When using QComboBox, items on the drop-down list have checkboxes, which is different from default behaviour and causes text cutting (see attached pictures):
With QDarkStyle:
base
dropdown
Default:
default
Is this behaviour intended? I noticed, that by deleting background-color lines in QWidget style I can get rid of checkboxes, but then obviously background colors are messed up.

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #132

The active element is now on **bold**, was the only thing I could manage to do. It seems the QComboBox implementation under the hood is funky.

Screen Shot 2019-04-30 at 12 42 32

On Spyder I still se a small artifact on the corners, but it must be related to something else?

Screen Shot 2019-04-30 at 12 45 05

Tested on OSX only

### CODE REVIEW

**Comment by goanpeca** [comment]
>Good job with this issue, but I can see the white corner in Linux Ubuntu yet. Ideas?

Nope, I tried it on a stand alone QComboBox example and it does not show so I am not sure what is causing it on Spyder.

File: qdarkstyle/style.qss [hunk 1]
```diff
@@ -1057,18 +1057,26 @@ QCommandLinkButton:disabled {
...
 QComboBox {
     border: 1px solid #32414B;
     border-radius: 4px;
     selection-background-color: #1464A0;
-    padding-top: 2px;     /* This fix  #103, #111*/
-    padding-bottom: 2px;  /* This fix  #103, #111*/
     padding-left: 4px;
     padding-right: 4px;
+    min-height: 1.5em;  /* This fixes  #103, #111 ? */
+    /* padding-top: 2px;     removed ro fix 132 */
```
[the remark this answers is not in the document]
Comment (goanpeca) [reply qdarkstyle/style.qss]: Fixed!

File: qdarkstyle/style.qss [hunk 2]
```diff
@@ -1082,12 +1090,25 @@ QComboBox:on {
     selection-background-color: #19232D;
 }
 
-QComboBox QAbstractItemView {
-    background-color: #19232D;
-    border-radius: 4px;
-    border: 1px solid #32414B;
-    selection-color: #148CD2;
-    selection-background-color: #32414B;
+/* Needed to remove indicator */
```
[the remark this answers is not in the document]
Comment (goanpeca) [reply qdarkstyle/style.qss]: Fixed!

File: qdarkstyle/style.qss [hunk 3]
```diff
@@ -1082,12 +1090,25 @@ QComboBox:on {
...
+QComboBox::indicator {
+    background-color:transparent;
+    selection-background-color:transparent;
+    color:transparent;
+    selection-color:transparent;
+}
+
+/* Needed to remove indicator */
```
[the remark this answers is not in the document]
Comment (goanpeca) [reply qdarkstyle/style.qss]: Fixed!

**Comment by goanpeca** [comment]
>Also, need to run script/process_qrc.py to update files. If you could not run, I run here :)

@dpizetta I would rather you run it for the moment, I do not have the whole toolchain installed. Thanks!

**Comment by dpizetta** [comment]
No problems :) I do it soon.

**Comment by goanpeca** [comment]
>No problems :) I do it soon.

Awesome! thanks :-)

**Pull request merged**